.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -109,11 +109,16 @@ jobs:
       image: ${{ fromJSON(needs.configure.outputs.params).build_image }}
       options: -u 1001:1001
     steps:
-    - uses: actions/checkout@v2
+    # GHA requires relative paths for actions. Copy actions from container root
+    # to CWD.
+    - run: cp -r /actions .
```

Comment:
How about a link instead of copying?

Reply:
I'm not sure what will happen if we unmount the container, and the link is still there. Nothing good, probably. Besides, it's a few kilobytes, so not a big deal.